redirects.yaml

```diff
@@ -674,7 +674,6 @@ packages/kde351.*/?: "https://releases.ubuntu.com/"
 partners/contact-us/?: "https://partner-portal.canonical.com/"
```

Comment:
We should add redirects from `/download/intel` and `/intel` to the new page on c.com